src/components/layouts/AddressOverview.vue

```diff
@@ -219,6 +217,21 @@ export default defineComponent({
         const unclaimedCashlinkCount = ref(0);
         const showUnclaimedCashlinkList = ref(false);
 
+        onMounted(() => {
+            // Using setTimeout just to make sure, this is executed at the end of stack call as sometimes, it adds the
+            // 'masked' class although it's not needed
+            setTimeout(() => maskAddress(), 1);
+
+            window.addEventListener('resize', maskAddress);
+        });
+        onUnmounted(() => window.removeEventListener('resize', maskAddress));
```

Comment:
The original commit (https://github.com/nimiq/wallet/commit/e3e6f3fd18ba9cb8baae0477f3c8376bd743670f) that introduced the `vue-responsive-components` dependency used a `ResizeObserver` on the `$address` element instead. While this here with a window resize listener works, I just want you to be aware that it has worse performance, as it fires on _all_ window resizes, whereas a ResizeObserver would only fire when that window resize actually changes the width of the element.